doc/rtd/howto/ubuntu_test_prerelease.rst

```diff
@@ -0,0 +1,50 @@
+.. _ubuntu_test_pre_release:
+
+Test pre-release cloud-init
+===========================
+
+Afterthe cloud-init team creates an upstream release, cloud-init will
+be released in the -proposed APT repository for a
+:ref:`period of testing<sru_testing>` Users are encouraged to test their
+workloads on this proposed release so that bugs can be caught and fixed prior
```

Comment:
No, this refers to the cloud-init release, not the repository / suite / distribution / pocket / whatever that it comes from.

Reply:
Noted - it's confusing though, given that we were also talking about -proposed. Can we change this to something like "on this pending release" or something similar to avoid the ambiguity?

Reply:
Agreed, thanks @s-makin!